GATK/4.1.3.0/SplitNCigarReads.nf

```diff
@@ -3,21 +3,18 @@ process SplitNCigarReads {
     tag {"GATK SplitNCigarReads ${sample_id}"}
     label 'GATK_4_1_3_0'
     label 'GATK_4_1_3_0_SplitNCigarReads'
-    clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.splitncigarreads_mem}" : ""
+    clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.splitncigarreads.mem}" : ""
     container = 'library://sawibo/default/bioinf-tools:gatk4.1.3.0'
     
     input:
       tuple sample_id, file(bam), file(bai)
-      file(genome_fasta)
-      file(genome_index)
-      file(genome_dict)
-
+   
     output:
       tuple sample_id, file("${sample_id}.split.bam"), file("${sample_id}.split.bai")
 
     script:
     """
-    gatk --java-options "-Xmx${task.memory.toGiga()-4}g -Djava.io.tmpdir=\$TMPDIR" \
+    gatk --java-options "-Xmx${task.memory.toGiga()-4}g -Djava.io.tmpdir=\$PWD" \

```

Comment:
Why? I think we should not use $PWD for tmp files.

Reply:
Will try that one

Reply:
I tend to agree with Robert but does this also work correctly on sge ? If not it would be a temporary fix

Reply:
For now, I reverted back to $TMPDIR but will do some testing and might come back if there are any issues.